runtime/src/main/java/org/corfudb/runtime/clients/SequencerClient.java

```diff
@@ -47,6 +47,32 @@ private static Object handleTokenResponse(CorfuPayloadMsg<TokenResponse> msg,
         return msg.getPayload();
     }
 
+    /**
+     * Handle an ACK response from the server.
+     *
+     * @param msg The ping request message
+     * @param ctx The context the message was sent under
+     * @param r   A reference to the router
+     * @return Always True, since the ACK message was successful.
+     */
+    @ClientHandler(type = CorfuMsgType.ACK)
```

Comment:
I don't think this is necessary, these messages are handled by baseclient already

Reply:
Apparently, the test router rails if these definitions are not here.

Reply:
It shouldn't unless you don't have a base client? The handlers are mapped by type, so declaring them twice would overwrite the previous registration...